### PR TITLE
AC Automated Test - LRAC-10147,LRAC-10152 and LRAC-10150 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10147
https://issues.liferay.com/browse/LRAC-10152
https://issues.liferay.com/browse/LRAC-10150

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by manuferreira — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+
+		task ("Block custom event") {
+			ACCustomEvents.blockCustomEvent(customEventName = "${customEventName}");
+		}
+
+		task ("Search for the name of the custom event that was blocked in custom event list") {
+			ACUtils.searchBar(searchItem = "${customEventName}");
+
+			ACUtils.viewNoResultsMessage(noResultMessage = "There are no items found.");
+		}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase:213]: Hi @ViniLopes20 , I believe this task is not needed. We already did this validation on the test `CanBlockCustomEvent`. What do you think?

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+			ACCustomEvents.createCustomEvent(
+				customEventName = "${customEventName}",
+				timeToSendCustomEvent = "1");
+		}
+
+		task ("Close sessions") {
+			ACUtils.closeAllSessionsAndWait();
+		}
+
+		task ("Login as an non-admin user and switch the property in AC") {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase:257]: hi @ViniLopes20 , please, fix this typo `Login as a non-admin`.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+			ACSettings.goToEvents();
+
+			ACNavigation.openItem(itemName = "Custom Events");
+		}
+
+		task ("Block custom event") {
+			ACCustomEvents.blockCustomEvent(customEventName = "${customEventName}");
+		}
+
+		task ("Logout AC and login as an non-admin user") {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase:315]: the same here @ViniLopes20 , please, fix this typo Login as a non-admin.

**Comment by alexcandidos** [comment]
Hi @ViniLopes20 , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Review by alexcandidos — APPROVED** [review]

**Pull request closed without merging**